shipyard_blueprints/shipyard-bp-utils/shipyard_bp_utils/artifacts.py

```diff
@@ -0,0 +1,66 @@
+import os
+import pickle
+from . import files
+from shipyard_templates import ShipyardLogger
+
+logger = ShipyardLogger.get_logger()
+
+
+class Artifact:
```

Comment:
this would replace the functionality that was in the log.py

the use would be `Artifact('vendor').response.create_pickle('vendor_response',response)`

Thoughts?

Reply:
I like it. Also open to other file types aside from pickle